apps/sensenet/cypress/integration/ui/logonavigation.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Logo Navigation Test', () => {
+  beforeEach(() => {
+    cy.login('admin')
```

Comment:
it is fine this way, just a tip:
login function's first parameter is 'admin' if not given explicitly, so cy.login() wold be enough in this case

Reply:
changed in latest commit